[REF] auto-fix conventions (+ manual fixes)

mdietrichc2c opened the pull request.

moylop260 committed:
[REF] auto-fix conventions

mdietrichc2c committed:
Fix pylint warnings

bwrsandman commented on l10n_ch_sepa/base_sepa/msg_sepa.py:
```diff
@@ -74,6 +74,7 @@ def _is_xsd_valid(self):
 
 class MsgSEPAFactory(object):
+
     """This class is a factory that creates SEPA message in order to allow
```
:-1: Is the newline before a doctring a positive change to style?

bwrsandman commented on l10n_ch_scan_bvr/tests/test_scan_bvr.py:
```diff
@@ -23,6 +23,7 @@
 
 class test_scan_bvr(common.TransactionCase):
+
     """ Test the wizard for bvr line scanning """
```
:-1: Is the newline before a doctring a positive change to style?

bwrsandman commented on l10n_ch_payment_slip_account_statement_base_completion/statement.py:
```diff
@@ -27,6 +27,7 @@
 
 class account_statement_completion_rule(orm.Model):
+
     """ Add a rule based on BVR Reference """
```
:-1: Is the newline before a doctring a positive change to style?

bwrsandman commented on l10n_ch_payment_slip/report/multi_report_webkit_html.py:
```diff
@@ -30,6 +30,7 @@
 class L10nCHReportWebkitHtmlMulti(report_sxw.rml_parse):
+
     """Report that output single BVR from invoice.
     This report is deprectated and will be merged
```
:-1: Is the newline before a doctring a positive change to style?

bwrsandman commented on l10n_ch_payment_slip/invoice.py:
```diff
@@ -227,6 +228,7 @@ def copy(self, cr, uid, inv_id, default=None, context=None):
 class AccountTaxCode(Model):
+
     """Inherit account tax code in order
     to add a Case code"""
```
:-1: Is the newline before a doctring a positive change to style?

bwrsandman commented on l10n_ch_payment_slip/invoice.py:
```diff
@@ -82,6 +82,7 @@ def _compute_bvr_ref(self, cr, uid, move_line, context=None):
 class AccountInvoice(Model):
+
     """Inherit account.invoice in order to add bvr
     printing functionnalites. BVR is a Swiss payment vector"""
```
:-1: Is the newline before a doctring a positive change to style?

bwrsandman commented on l10n_ch_payment_slip/company.py:
```diff
@@ -23,6 +23,7 @@
 
 class ResCompany(Model):
+
     """override company in order to add bvr vertical and
```
:-1: Is the newline before a doctring a positive change to style?

bwrsandman commented on l10n_ch_payment_slip/bank.py:
```diff
@@ -21,6 +21,7 @@
 class ResPartnerBank(orm.Model):
+
     """
     Inherit res.partner.bank class in order to add swiss specific fields
```
:-1: Is the newline before a doctring a positive change to style?

bwrsandman commented on l10n_ch_lsv_dd/model/move_line.py:
```diff
@@ -23,6 +23,7 @@
+
     '''
     Use hooks to add bvr ref generation if account is IBAN and has LSV
     identifier
```
:-1: Is the newline before a doctring a positive change to style?

bwrsandman commented on l10n_ch_dta/wizard/create_dta.py:
```diff
@@ -484,10 +491,12 @@ def init_local_context(self):
 class record_gt890(record):
+
     """Implements Total Record of DTA file
     if behaves like a account payment order
```
:-1: Is the newline before a doctring a positive change to style?

bwrsandman commented on l10n_ch_dta/wizard/create_dta.py:
```diff
@@ -371,6 +377,7 @@ def init_local_context(self):
 
 class record_gt836(record):
+
     """Implements iban record"""
```
:-1: Is the newline before a doctring a positive change to style?

bwrsandman commented on l10n_ch_dta/wizard/create_dta.py:
```diff
@@ -286,9 +290,11 @@ def validate_global_context_dict(self):
 class record_gt827(postal_record):
+
     """
     Swiss internal (bvpost and bvbank) record implemetation
```
:-1: Is the newline before a doctring a positive change to style?

bwrsandman commented on l10n_ch_dta/wizard/create_dta.py:
```diff
@@ -171,6 +174,7 @@ def validate_global_context_dict(self):
 
 class record_gt826(postal_record):
+
     """ BVR record implementation"""
```
:-1: Is the newline before a doctring a positive change to style?

bwrsandman commented on l10n_ch_dta/wizard/create_dta.py:
```diff
@@ -139,10 +140,12 @@ def generate(self):
 class postal_record(record):
+
     """Class that propose common
     knowledge for all postal account type
```
:-1: Is the newline before a doctring a positive change to style?

bwrsandman commented on l10n_ch_dta/wizard/create_dta.py:
```diff
@@ -78,6 +78,7 @@ def tr(string_in):
 class record(object):
+
     """Abstract class that provides common
     knowledge for any kind of post/bank account
```
:-1: Is the newline before a doctring a positive change to style?

bwrsandman commented on l10n_ch_dta/bank.py:
```diff
@@ -21,6 +21,7 @@
 class ResPartnerBank(orm.Model):
+
     """
     Inherit res.partner.bank class in order to add swiss specific fields
```
:-1: Is the newline before a doctring a positive change to style?

bwrsandman commented on l10n_ch_credit_control_payment_slip_report/credit_control_printer.py:
```diff
@@ -23,6 +23,7 @@
 class CreditControlPrinter(orm.TransientModel):
+
     """Print lines"""
     _inherit = 'credit.control.printer'
```
:-1: Is the newline before a doctring a positive change to style?

bwrsandman commented on l10n_ch_credit_control_payment_slip_report/credit_control_communication.py:
```diff
@@ -37,12 +38,13 @@ def _generate_report_bvr(self, cr, uid, line_ids, context=None):
+
     """We define a new parser because this report take move line
        In parameter insted of an invoice, so the function get_obj_reference
        return directly ids"""
```
:-1: Is the newline before a doctring a positive change to style?

bwrsandman commented on l10n_ch_credit_control_payment_slip_report/credit_control_communication.py:
```diff
@@ -25,6 +25,7 @@
 class CreditCommunication(orm.TransientModel):
+
     """Shell class used to provide a base model to email template
        and reporting.
```
:-1: Is the newline before a doctring a positive change to style?

bwrsandman commented on l10n_ch_credit_control_payment_slip_report/account.py:
```diff
@@ -22,6 +22,7 @@
 
 class account_move_line(orm.Model):
+
     """Overrride BVR amount to take in account dunning fees"""
```
:-1: Is the newline before a doctring a positive change to style?

bwrsandman commented on l10n_ch_base_bank/bank.py:
```diff
@@ -144,6 +145,7 @@ def name_search(self, cursor, uid, name, args=None, operator='ilike',
+
     """
     Inherit res.partner.bank class in order to add swiss specific fields
     and state controls
```
:-1: Is the newline before a doctring a positive change to style?

bwrsandman commented on l10n_ch_base_bank/bank.py:
```diff
@@ -53,6 +53,7 @@ def _check_5_pos_postal_num(self, number):
 
 class Bank(orm.Model, BankCommon):
+
     """Inherit res.bank class in order to add swiss specific field"""
```
:-1: Is the newline before a doctring a positive change to style?

bwrsandman commented on l10n_ch_account_statement_base_import/parser/v11_file_parser.py:
```diff
@@ -32,6 +32,7 @@ def is_total_record_type4(test):
 class V11FileParser(BankStatementImportParser):
+
     """
     A v11 file parser.
```
:-1: Is the newline before a doctring a positive change to style?

bwrsandman commented on l10n_ch_account_statement_base_import/parser/postfinance_file_parser.py:
```diff
@@ -32,9 +32,11 @@
+
     """
     Parser for XML Postfinance Statements (can be wrapped in a tar.gz file)
     """
```
:-1: Is the newline before a doctring a positive change to style?

gurneyalex committed:
use raw string in regexp

gurneyalex committed:
sort the lines of the dictionary

I sorted 2 blocks separately, according to the comments.
The goal is to ease finding duplicate keys.

gurneyalex committed:
remove 2 duplicate entries in the dictionary

gurneyalex committed:
fix blank lines

gurneyalex committed:
break circular imports

gurneyalex commented:
:+1: with my fixes applied

lepistone commented:
:+1:

nbessi commented:
:+1:
